sample/src/main/kotlin/com/slack/circuit/sample/home/HomePresenter.kt

```diff
@@ -80,21 +101,29 @@ class HomePresenter
 @AssistedInject
 constructor(
   @Assisted private val navigator: Navigator,
-  petListPresenterFactory: PetListPresenter.Factory
+  petListFilterPresenterFactory: PetListFilterPresenter.Factory
 ) : Presenter<HomeScreen.State, HomeScreen.Event> {
-  private val petListPresenter = petListPresenterFactory.create(navigator)
+  private val petListFilterPresenter = petListFilterPresenterFactory.create()
 
-  @SuppressLint("FlowOperatorInvokedInComposition")
   @Composable
   override fun present(events: Flow<HomeScreen.Event>): HomeScreen.State {
-    val homeNavState =
-      homeNavPresenter(events.filterIsInstance<HomeScreen.Event.HomeEvent>().map { it.event })
-    val petListState =
-      petListPresenter.present(
-        events.filterIsInstance<HomeScreen.Event.PetListEvent>().map { it.event }
-      )
+    val rememberHomeNavState = remember {
+      events.filterIsInstance<HomeScreen.Event.HomeEvent>().map { it.event }
+    }
+    val homeNavState = homeNavPresenter(rememberHomeNavState)
+
+    val rememberPetListFilterState = remember {
```

Comment:
nit: this is an event flow, not a state

Reply:
Addressed here, https://github.com/slackhq/circuit/pull/101/commits/89de7f6d29b498d957c17c4ec559be364ecd09c1.